types/mparticle__web-sdk/index.d.ts

```diff
@@ -3,6 +3,45 @@ import { Batch } from "@mparticle/event-models";
 // Placeholder for Dictionary-like Types
 export type Dictionary<V = any> = Record<string, V>;
 
+// Rokt Manager Types
+export interface RoktPartnerAttributes {
+    [key: string]: string | number | boolean | undefined | null;
+}
+
+export interface RoktPartnerExtensionData<T> {
+    [extensionName: string]: T;
+}
+
+export interface RoktSelectPlacementsOptions {
+    attributes: RoktPartnerAttributes;
+    identifier?: string;
+}
+
+export type RoktPlacement = Record<string, unknown>;
+
+export interface RoktSelection {
+    close: () => void;
+    getPlacements: () => Promise<RoktPlacement[]>;
+}
+
+export interface RoktLauncher {
+    selectPlacements: (options: RoktSelectPlacementsOptions) => Promise<RoktSelection>;
+    hashAttributes: (attributes: RoktPartnerAttributes) => Promise<Record<string, string>>;
+}
+
+export interface RoktMessage {
+    methodName: string;
+    payload: any;
+}
+
+export interface RoktOptions {
+    sandbox?: boolean;
+    launcherOptions?: RoktLauncherOptions;
+    domain?: string;
+}
```

Comment:
Is this necessary?  I think this is only used internally, and doesn't need to be exported for a developer to use, right?